tests/pyreverse/test_diadefs.py

```diff
@@ -50,7 +54,8 @@ def HANDLER(default_config: PyreverseConfig) -> DiadefsHandler:
 
 @pytest.fixture(scope="module")
 def PROJECT(get_project):
-    return get_project("data")
+    with _test_cwd(TESTS):
```

Comment:
We should use ``tmpdir.as_cwd()`` (until we find something better ?) see https://github.com/PyCQA/pylint/pull/7173

Reply:
Replied at https://github.com/PyCQA/pylint/pull/7173#issuecomment-1182582877.

Reply:
Resolving as we determined that ``_test_cwd`` is actually the best way to do this.